java/cdk/src/main/java/sleeper/cdk/stack/bulkimport/EmrServerlessBulkImportStack.java

```diff
@@ -0,0 +1,245 @@
+        String instanceId = instanceProperties.get(ID);
+        IVpc vpc = Vpc.fromLookup(this, "VPC",
+                VpcLookupOptions.builder().vpcId(instanceProperties.get(VPC_ID)).build());
+
+        SecurityGroup securityGroup = SecurityGroup.Builder
+                .create(this, String.join("-", "SG", "sleeper", instanceId, "EMR-Serverless"))
+                .description("Security Group used by EMR Serverless").vpc(vpc).build();
+        // Save ID
+        return securityGroup.getSecurityGroupId();
+    }
+
+    private IRole createEmrServerlessRole(Construct scope, InstanceProperties instanceProperties) {
+        String instanceId = instanceProperties.get(ID);
+        Role role = new Role(this, "EmrServerlessRole", RoleProps.builder()
+                .roleName(String.join("-", "sleeper", instanceId, "EMR-Serverless-Role"))
+                .description("The role assumed by the Bulk import EMR Serverless Application")
+                .managedPolicies(
+                        List.of(createEmrServerlessManagedPolicy(scope, instanceProperties)))
+                .assumedBy(new ServicePrincipal("emr-serverless.amazonaws.com")).build());
+
+        instanceProperties.set(BULK_IMPORT_EMR_SERVERLESS_CLUSTER_ROLE_ARN, role.getRoleArn());
+        return role;
+    }
+
+    private ManagedPolicy createEmrServerlessManagedPolicy(Construct scope,
+            InstanceProperties instanceProperties) {
+        // See https://docs.aws.amazon.com/emr/latest/EMR-Serverless-UserGuide/getting-started.html
+        String instanceId = instanceProperties.get(ID);
+        String bulkImportBucket = instanceProperties.get(BULK_IMPORT_BUCKET);
+        String ingestSourceBucket = instanceProperties.get(INGEST_SOURCE_BUCKET);
+        String configBucket = String.join("-", "sleeper", instanceId, "config");
+
+        List<String> buckets = Stream.of("arn:aws:s3:::" + bulkImportBucket,
+                "arn:aws:s3:::" + bulkImportBucket + "/*",
+                "arn:aws:s3:::" + ingestSourceBucket,
+                "arn:aws:s3:::" + ingestSourceBucket + "/*",
+                "arn:aws:s3:::" + configBucket,
+                "arn:aws:s3:::" + configBucket + "/*").collect(Collectors.toList());
+
+        List<String> tables = new ArrayList<>();
+        String base = "arn:aws:dynamodb:" + instanceProperties.get(REGION) + ":" + instanceProperties.get(ACCOUNT) + ":table/";
+        tables.add(base + String.join("-", "sleeper", instanceId, "ingest-job-status"));
+
+        Utils.getAllTableProperties(instanceProperties, scope).forEach(tableProperties -> {
+            String table = String.join("-", "sleeper", instanceId, "table", tableProperties.get(TABLE_NAME));
+            String bucketArn = "arn:aws:s3:::" + table;
+            buckets.add(bucketArn);
+            buckets.add(bucketArn + "/*");
+            tables.add(base + table + "-partitions");
+            tables.add(base + table + "-active-files");
+        });
```

Comment:
We could pass in BulkImportBucketStack, TableStack and IngestStatusStoreStack to avoid needing to manually build the resource ARNs here. Look at how the EC2 role is built in CommonEmrBulkImportStack.